test/TestNamespaceTracker.hs

```diff
@@ -333,8 +333,7 @@ trackerHelper' owners registered as =
     mapPack <$> gogo as' i (trackerProto <-> fakeServer)
   where
     trackerProto = namespaceTrackerProtocol owners registered
-    i = trace "piss" j
```

Comment:
Hmm, perhaps we should squash some of this history? :-/ I think I might have been getting a _tad_ frustrated with this before I gave up.

Reply:
Erm, that was me getting annoyed at the fact that I couldn't get the trace to work where I wanted. But, maybe, there's a fair bit of back and forth and WIP.

Reply:
:tada: not me! Yeah, not quite sure how to neaten it, but we really don't need all 76 commits! (Unless you want our github stats to look beefy :cow2:)